README.md

```diff
@@ -63,34 +63,67 @@ Objectives:
 
-### Deprecated
+#### Deprecated
 
 The component is planned to be removed in a future version and no further support will be provided. Note that new issues will likely not be worked on. When a component enters "deprecated" mode, it is expected to exist for at least two minor releases. See the component's readme file for more details on when a component will cease to exist.
 
+### Metric stability levels
+
+Receivers that produce metrics should declare a stability level for each metric. A for each metric may not exceed the The status for each metric should be listed in the receiver's `documentation.md` file according to the following definitions:
+
+#### In development
+
+The metric may be documented but not available, or available but subject to change at any time. Bugs and performance issues should be reported, but it is likely that the component owners might not give them much attention. Your feedback is still desired. The metric should not be used in production.
+
+#### Alpha
+
+The component is ready to be used for limited non-critical workloads and the authors of this component would welcome your feedback. Bugs and performance problems should be reported, but component owners might not work on them right away. The configuration options might change often without backwards compatibility guarantees.
+
+#### Beta
+
+Same as Alpha, but the configuration options are deemed stable. While there might be breaking changes between releases, component owners should try to minimize them. A component at this stage is expected to have had exposure to non-critical production workloads already during its **Alpha** phase, making it suitable for broader usage.
+
+#### Stable
+
+The component is ready for general availability. Bugs and performance problems should be reported and there's an expectation that the component owners will work on them. Breaking changes, including configuration options and the component's output are not expected to happen without prior notice, unless under special circumstances.
+
+#### Deprecated
+
+The component is planned to be removed in a future version and no further support will be provided. Note that new issues will likely not be worked on. When a component enters "deprecated" mode, it is expected to exist for at least two minor releases. See the component's readme file for more details on when a component will cease to exist.
+
+### Stability level progression
+
+Comonents and metric stability levels should progress according to the following guidelines:
+- A component that produces metrics should produce at least one metric of the same stability level. For example, a Stable component should produce at least one Stable metric.
```

Comment:
Why? If we did not have metrics in a stable component, and start adding them now? How do I get to stable directly?

Reply:
My intention here is to address primarily how it "should" be. Then we would take steps to get there.

As to the reasoning here, this just seems necessary in order to give meaning to the component status. I would question whether a scraper can be stable if it produces no stable metrics.

Reply:
I was mostly trying to address scrapers, but didn't use the term because I'm not sure it's a user-facing term. (I said receivers that produce metrics instead, but maybe this isn't specific enough.)

That said, I think all of this would apply to own telemetry as well, with the exception of this line.

Reply:
How is this different than "receiver that produce logs" or "spans" should we actually call this "telemetry"?

I would probably have a different term for "own observability telemetry" vs "telemetry on behalf of others" :))

Reply:
> I would probably have a different term for "own observability telemetry" vs "telemetry on behalf of others" :))

Makes sense. I think "own telemetry" would be clear and is already used elsewhere (eg OpAMP). Perhaps "scraped telemetry" then for the other term. 

> How is this different than "receiver that produce logs" or "spans" should we actually call this "telemetry"?

The way I see it is that metrics have named instances, each of which has its own detailed data model. Maybe I'm overlooking something, but I don't think logs and spans have an equivalent really.

In any case, I can try to organize this PR to leave room for future stability guidelines for other signals, but I only have detailed thoughts for metrics at this point. What do you think of organizing it like this:

```
## Metric stability levels

### "Own metrics" vs "Scraped metrics"

// TODO describe the difference and give a couple examples

The following levels and guidelines apply to "own metrics" and "scraped metrics", except where otherwise noted.

...

### Scraped metrics

Receivers whose sole purpose is to scrape metrics (ie. "scrapers") should produce at least one metric of the same stability level. For example, a Stable component should produce at least one Stable metric.
```

Then if someone can reuse some of these guidelines for logs and/or spans, they can extract them into a "telemetry" header.